metapool/scripts/seqpro_mf.py

```diff
@@ -57,6 +60,13 @@ def format_preparation_files_mf(run_dir, mapping_file, output_dir):
                            'tm1000_8_tool', 'tm300_8_tool', 'tm50_8_tool',
                            'water_lot', 'well_description', 'well_id'])
 
+        if verbose:
+            project_name = remove_qiita_id(project)
+            # assume qiita_id is extractable and is an integer, given that
+            # we have already passed error-checking.
+            qiita_id = project.replace(project_name + '_', '')
+            print("%s (%s)" % (abspath(fp), qiita_id))
```

Comment:
I don't think we need the `()`, right? and it might be better to do `qiita_id filepath` so we know that the qiita_id is the first thing and the rest is the filepath ... AKA no extra parsing.

Reply:
ok.